### ISSUE TITLE
New Features

### ISSUE THREAD

**Issue by johntom** [nihgwu/hexo-hey#35]
[Image-Sample.txt](https://github.com/nihgwu/hexo-hey/files/679867/Image-Sample.txt)

Hi,
I tried to fork as add 2 fields to the metadata section of a post, without success. In my case, I use the following  https://github.com/ptsteadman/hexo-theme-corporate.git which when opened will show additional fields thumbnail and lede for each post.
If possible I would like those feilds added to the edit/insert mode of a post. It would also be awsome to add an uploaded image the thumbnail section along with the body. The following is an example 
Image-Sample.md also attached as txt file.
title: Image Sample
tags:
  - image
  - test
categories:
  - hexo
date: 2016-09-03 13:30:00
thumbnail: http://localhost:4000/images/1483213724160.jpg
lede: "Hexo makes incredible static websites. Hexo Hey gives programmers, UX designers and end users the ability to add posts and allows for drag and drop images
 that really stand out."
---
## Upload an image using drag and drop
# Abstract piece # 3
![image](http://localhost:4000/images/1483213724160.jpg)
Thanks,
John

**Comment by nihgwu** [comment]
I know your concern, I think the easiest way for you to achieve it is write the mete section in the body directly
I'm planning to refactor this project this month, thanks for your interest

**Comment by johntom** [comment]
i think I tried the body but since it's past the meta section it does not get picked up. I'm willing to contribute some funds if interested. I was using hexo-admin and find your solution better fits my needs.
FYI, while I still have many angular 1.x apps I've moved on to Aurelia and react-native for all new stuff.
Thanks and Happy New Year
John

**Comment by nihgwu** [comment]
I've moved on to react and react-native 😄  so I'm planning to rewrite this in react as I can't bare ng1 anymore.

**Comment by johntom** [comment]
Aurelia is worth a look, more of complete framework than react which is why I use it for my desktop apps. I live in nyc and goto all Angular meetups and felt NG2 too verbose and difficult. Aurlia follows standards and is quite fun to work with.
John

**Comment by johntom** [comment]
Hi,
Just checking on the status of the refactor if its still something on your plate.
Thanks,
John

**Comment by nihgwu** [comment]
yes, I will start after update one of my app, should be by next week, thanks for your remind

**Comment by johntom** [comment]
Just checking to see if you still have plans with this repo
